CHANGELOG.md

```diff
@@ -17,11 +17,16 @@ the detailed section referring to by linking pull requests or issues.
 #### Added
 
 - Added login polling to prevent auto-logout when page is open.
+- Added handling of the additional properties in the case of assets
```

Comment:
Wording "the additional properties" is vague

- Now showing all asset properties for assets and contract offers via an "Additional Properties" section.